### ISSUE TITLE
Add labeled items for accessibility

### ISSUE THREAD

**Issue by OzancanKaratas** [deltachat/deltachat-desktop#870]
No Windows builds. So I cannot give any information.

NVDA is a free and open source screen reader for Windows. We are using it to use the computers. Visually impaired users can access to the Javascript applications such as Skype.

NVDA APIs is used for manage and read messages from external applications. If Delta Chat's continuous integrations does a build for Windows, please interest this issue. I'm ready to help and send feedback for blinds.

[NVDA repository](https://github.com/nvaccess/nvda)

Note: NVDA APIs written with C++. We should work for Javascripts codes.

**Comment by Simon-Laux** [comment]
Great Idea with including screen reading...
But I see a few points against integrating NVDA api in deltachat desktop electron:
1. We already had/have difficulties with deltachat-node building, adding another native dependency won't help.
2. we should add #488 first in my opinion (Using a mouse when blind sounds no fun, but on the other hand I don't know how you navigate a computer when blind at all)
3. Libraries for deltachat-desktop (electron edition) should be cross platform (linux,macOS,windows).

I would love a good feature proposal on the forum (https://support.delta.chat/) as accessibility is a topic that can be interesting for other platforms too.

PS: We could maybe start a new speech-based deltachat client with the core that has a comandline interface for input and speaks with the user. Let's talk about such ideas on the forum https://support.delta.chat/

**Comment by OzancanKaratas** [comment]
I saw the button labeling process is better than adding NVDA API. Unlabeled buttons bigger problem. I think we should label the DeltaChat buttons. My screen reader does not read the several buttons, just announces button. I must select the button to see what happens.

**Comment by Simon-Laux** [comment]
I guess its the buttons that have an icon instead of text, I'll try to label them with the area-label attribute.

**Comment by Simon-Laux** [comment]
I think this is done now, please reopen or create a new issue when new labeling or accessibility issues arise.
The accessibility strings will get into transifex when we do the next "putting all new language strings into transifex" task.
